Remotion/Data/DomainObjects.UnitTests/Persistence/StorageProviderManagerTest.cs

```diff
@@ -100,8 +101,8 @@ public void GetMandatory_WithProviderInstanceNotPartOfStorageSettings ()
     [Test]
     public void GetMandatory_WithProviderID_ReturnsSameInstanceTwice ()
```

Comment:
please add test that shows that IReadOnlyMgr and IMgr return the same instance for the same ID.